src/persist/src/postgres.rs

```diff
@@ -17,7 +17,7 @@ use async_trait::async_trait;
 use tokio::sync::Mutex;
 use tokio::task::JoinHandle;
 use tokio_postgres::types::{to_sql_checked, FromSql, IsNull, ToSql, Type};
-use tokio_postgres::{Client as PostgresClient, NoTls};
+use tokio_postgres::{Client as PostgresClient, IsolationLevel, NoTls, Transaction};
 
 use mz_ore::task;
 
```

Comment:
github won't let me comment on the actual line, but do we still need the `SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL SERIALIZABLE` in SCHEMA after this?

Reply:
removed.

Reply:
hopefully it wasn't structural! 😬

Reply:
thanks for the new nightmare dan

(i believe removing is fine because all ops are wrapped in a serializable txn now)